### ISSUE TITLE
Should land latex/goo creatures be spawned during day time?

### ISSUE THREAD

**Issue by XIONGCHUMMO** [LtxProgrammer/Changed-Minecraft-Mod#1145]
I've tried v0.14. But I found it's much more difficult to prevent being transformed. As for me, I do prefer such creatures only can be spawned in night like before.

**Comment by Anonogon** [comment]
They most certainly should not spawn during the day. Not only does it make survival gameplay incredibly difficult, but it also makes villages almost impossible to maintain.

**Comment by Y1rd** [comment]
Yeah, it needs to be nerfed quite a bit at least. They spawn too much during day.

**Comment by Y1rd** [comment]
> Yeah, it needs to be nerfed quite a bit at least. They spawn too much during day.

Or at least add some configuration option to change spawn chance during daytime.

**Comment by gengyoubo** [comment]
#1140

**Comment by gengyoubo** [comment]
![屏幕截图 2024-11-05 195044](https://github.com/user-attachments/assets/93f1e91c-8832-4c7e-9683-8a4baf69f8ec)

**Comment by gengyoubo** [comment]
"Why is the latex refresh rate so high in the middle of the day?"
  “I didn't go more than a few steps before I met one, and the one playing limit went straight to GG."

> ![屏幕截图 2024-11-05 195044](https://private-user-images.githubusercontent.com/113955003/383114843-93f1e91c-8832-4c7e-9683-8a4baf69f8ec.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA4MDQxNTUsIm5iZiI6MTczMDgwMzg1NSwicGF0aCI6Ii8xMTM5NTUwMDMvMzgzMTE0ODQzLTkzZjFlOTFjLTg4MzItNGM3ZS05NjgzLThhNGJhZjY5ZjhlYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwNVQxMDUwNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OWM5MzljYmYyNzRmYjM0OWY0OTA3NzAxOGNkNmU5YzFlMzJmOTYyMjQ4NGRjMjI3MTBjZWY1Y2FkYjg3NWE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.VtWm_5oEAjbM6sKw1Wr9lOwGQ9GkXwvF1_w_FlHS1e4)
